master/buildbot/test/__init__.py

```diff
@@ -119,3 +119,6 @@
 
 # FIXME: needs to be sorted out (#3666)
 warnings.filterwarnings('ignore', "Invalid utf8 character string")
+
+# twisted.compat.execfile is using 'U' https://twistedmatrix.com/trac/ticket/9023
```

Comment:
@rodrigc next time please add exceptions for those words to https://github.com/buildbot/buildbot/blob/master/common/code_spelling_ignore_words.txt